src/vs/workbench/contrib/terminal/browser/terminalIconPicker.ts

```diff
@@ -74,8 +80,12 @@ export class TerminalIconPicker extends Disposable {
 			if (hoverWidget) {
 				this._register(hoverWidget);
 			}
-			this._iconSelectBox.layout(dimension);
+			this._iconSelectBox.layout(new Dimension(486, 260));
 			this._iconSelectBox.focus();
+			// Force a rerender to make the position correct
+			setTimeout(() => {
+				window.dispatchEvent(new Event('resize'));
+			}, 10);
```

Comment:
Not sure what's happening here but in the PR right now with this resize event it will have 2 scroll bars:

![image](https://github.com/user-attachments/assets/e7aa5f8f-12f9-4e80-b39a-3721f32ddf93)

Removing the synthetic resize event we see the right size and scroll bar, but it's not attached to the target anymore:

![image](https://github.com/user-attachments/assets/414d2d5d-fb88-4cad-867e-9918ddf260d7)

🤔

Reply:
I can't seem to get the double scroll bars, but I do agree that the resize event is an ugly solution. I found what caused the problem and "fixed" it. There was a check for whether the hover was going beyond the window, but it seems to no account for something and falsely deemed the hover to be outside the window and thus moving it away from the target. When I subtracted the height of the hover in that comparison (or removed the check entirely) the issue of the placement disappeared.